packages/snaps-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.1.0]
+### Added
+- Add static file API ([#1836](https://github.com/MetaMask/snaps/pull/1836))
+  - This adds a `snap_getFile` method, which Snaps can use to load files.
+- Add `origin` parameter to `snapInstalled` and `snapUpdated` events ([#1867](https://github.com/MetaMask/snaps/pull/1867))
+
+### Changed
+- Improve error handling ([#1841](https://github.com/MetaMask/snaps/pull/1841))
+  - Snaps can now throw a `SnapError`, without causing the Snap to crash.
+- Bump `tar-stream` from `^2.2.0` to `^3.1.6` ([#1853](https://github.com/MetaMask/snaps/pull/1853))
+- Rename `rpc-methods` to `snaps-rpc-methods` ([#1864](https://github.com/MetaMask/snaps/pull/1864))
```

Comment:
Not sure if it makes sense to include this.

Reply:
I did remove it in a bunch of other changelogs, I can remove it here too